src/components/Auth/auth-form.tsx

```diff
@@ -18,9 +18,20 @@ function AuthForm() {
   const [signInMessage, setSignInMessage] = useState<string>('');
   const [loadingMessage, setLoadingMessage] = useState('');
   const { data: session, status } = useSession();
+  const userIdfentifier = (session?.user as any)?.id;
```

Comment:
variable name typo

Reply:
good catch

Reply:
Thanks for catching that! I have corrected the typo